wallit/init.go

```diff
@@ -107,8 +109,27 @@ func NewWallit(
 
 	// deal with incoming height
 	go w.HeightHandler(incomingBlockheight)
-
-	return &w
+	if !hookFail {
+		switch p.Name {
```

Comment:
I think you can do something like 
```
return &w, p.HDCoinType, nil
```
instead

Reply:
Yeah, was about to get to this, got stuck at the jenkins file thing. updating, thanks!